docs/targets/solana.rst

```diff
@@ -217,18 +217,42 @@ the provided seeds, along with a seed bump. See the Solana documentation on
         }
     }
 
-Using spl-token
-_______________
 
-`spl-token <https://spl.solana.com/token>`_ is the solana native way of creating tokens, minting, burning and
-transfering token. This is the Solana equivalent of
+
+Solana Library
+______________
+
+In Solang's Github repository, there is a folder called ``solana-library``. It contains libraries for Solidity contracts
+to interact with Solana specific instructions. Currently, there are two libraries there. One for SPL tokens and another
+for Solana's system instructions. In order to use those functionalities, one must copy the correspondent library
+file to one's project folder and import the library.
```

Comment:
```suggestion
In Solang's Github repository, there is a directory called ``solana-library``. It contains libraries for Solidity contracts
to interact with Solana specific instructions. Currently, there are two libraries there: one for SPL tokens and another
for Solana's system instructions. In order to use those functionalities, copy the correspondent 
file to one's project and import it.
```

Reply:
`copy the correspondent file to one's project`. The usage of articles must maintain parallelism. If I use `one must`, the possessive must be `one's`. If I use the imperative form `copy`, the possessive pronoun must be `your`. Do you have any preference over one form?